FiMAdminApi/Endpoints/TruckRoutesEndpoints.cs

```diff
@@ -94,5 +95,7 @@ public class EditTruckRoute
         public required string Name { get; set; }
 
         public List<Guid>? EquipmentIds { get; set; }
+
+        public StreamingConfig? Streaming_Config { get; set; }
```

Comment:
```suggestion
        public StreamingConfig? StreamingConfig { get; set; }
```

(Would require matching UI change)

Reply:
corrected and updated in UI